printer_mods/al3ph/led_mount/README.md

```diff
@@ -0,0 +1,5 @@
+Led holder for WS2812 type LED strings, includes channel for data line to loop back for second strip.
+
+Mounts on top extrusion, using 2 M3 bolts per strip, two strips required per side, (so they can be printed on V0).
+
+Requires support, if not printed vertically.

```

Comment:
Did you print it vertically?  It seems to have a very small base and I'm concerned that it will warp and detach from the bed if printed vertically without a brim.  If it requires a brim to be printed vertically, then one needs to be built in to the model.  If you want people to print it horizontally with supports, then supports must be built in to the model.

Reply:
Supports added, and orientation changed.